components/file_preview_modal/file_preview_modal_image_controls/file_preview_modal_image_controls.scss

```diff
@@ -0,0 +1,70 @@
+.image-controls {
+    position: relative;
+    display: flex;
+    height: 48px;
+    align-items: center;
+    justify-content: center;
+    background: #222224;
+    color: white;
+
+    &__dropdown {
+        height: 32px;
+        padding: 0 23px 0 12px;
+        border: none;
+        margin-left: 16px;
+        appearance: none;
+        background-color: rgba(255, 255, 255, 0.04);
+        background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='none' stroke='white' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3e%3cpolyline points='6 9 12 15 18 9'%3e%3c/polyline%3e%3c/svg%3e");
```

Comment:
is this needed? could we just use the respective compass-icon for it?

Reply:
I think the issue with using compass-icons is that it doesn't ship with the raw svg files and this needs to be used as a background image since it's a native html select element. I tried using scss extend `@extend .icon-caret-down` but webpack is giving me a `selector not found` sass error

In the meantime I cleaned up the css to import the icon from a static svg file.

Reply:
In this case I would prefer we not use an extra svg-file and keep the old implementation. Haven't worked with `select` elements in a while, but now I remember as well ... We are normally using `react-select` when we need something like this or just build it custom, but only if the other option is not working for this.

In this specific case I would be ok if we use the native select since we only use it here and nowhere else.

Reply:
@michelengelen by old implementation do you mean the svg code directly in the css or are you talking about another implementation for the select?

Reply:
I meant you could just use the implementation you had before with the svg string in the background property